lib/utils/plugins.js

```diff
@@ -69,6 +69,7 @@ const clearModulesCache = () => {
 };
 
 const getPluginName = path => window.require('path').basename(path);
+const getPluginVersion = path => window.require(window.require('path').resolve(path, 'package.json')).version;
```

Comment:
You mean like that https://github.com/zeit/hyper/blob/1.3.3/app/plugins.js#L129-L142
😰 
😄